### ISSUE TITLE
Xbox Wireless Controller in Bluetooth inputs trigger twice on a focused window

### ISSUE THREAD

**Issue by pho** [godotengine/godot#41684]
**Godot version:**
3.2.2-mono

**OS/device including version:**
Windows 10 Pro Version 1909
Xbox One controller via bluetooth

**Issue description:** 
A single joypad button press:
 - triggers one event from input device 0 if the game window doesnt have focus (as expected)
 - but triggers TWO events from devices 0 and 1, when the window is focused

There is also a shift in the device index:
- With the window unfocused, the controller is device 0
- With focus, the mouse is device 0 and the controller triggers both 0 and 1

See https://gfycat.com/physicalfortunatefinwhale

**Steps to reproduce:**
Check out the attached project, its just a func _input() with a print

**Minimal reproduction project:**
[BugReport.zip](https://github.com/godotengine/godot/files/5158304/BugReport.zip)

**Comment by KoBeWi** [comment]
Oh I had the same issue, with the same setup: #35903
But it's elusive to reproduce, because I tested with the same controller on another W10 laptop and it didn't happen.

**Comment by KoBeWi** [comment]
So this started happening to me again, with new XBox One S controller and via cable. Seems not consistent anymore though, sometimes there is one event, sometimes 2. My workaround is checking input inside `_process()` instead `_input()` bruh (I'm using actions).

**Comment by MOSN97** [comment]
I'm having this same problem in version 3.3.2 using an Xbox Series controller. It's rather problematic since my game is multiplayer played with several controllers.
I've done some testing and when the window is focused it seems the first controller counts as both device 0 and 1 and if a new controller is added it will be device 2.
However if the game is started while two controllers are connected the first controller will count as 0 and 2 while the second will count as 1 like it's supposed to.

**Comment by Gongure** [comment]
I'm having the same issue.

**Comment by akien-mga** [comment]
Fixed by #71784, at least for Xbox Series controllers.

If anyone has an Xbox One controller and could check if the bug is solved or not in the next beta (4.0 beta 15), that would be great!

**Comment by Calinou** [comment]
For future reference:

- I tested https://github.com/godotengine/godot/pull/71784 on Windows 11 22H2 with an Xbox One V4 controller in Bluetooth.
- With stock 4.0.beta14, in the joypads demo, I only see one controller (`XInput Gamepad __XINPUT_DEVICE_`) and it works correctly. Therefore, I can't reproduce https://github.com/godotengine/godot/issues/41684 in the first place.
- Same with an Xbox One V3 controller (also in Bluetooth); I only see it once. It also appears as `XInput Gamepad __XINPUT_DEVICE_` in the joypads demo.
- I've tested the pull request and can confirm it doesn't regress on this aspect.
